mitmproxy/console/options.py

```diff
@@ -91,6 +91,12 @@ def __init__(self, master):
                     lambda: master.server.config.check_tcp,
                     self.tcp_proxy
                 ),
+                select.Option(
+                    "Don't Verify Upstream Certs",
+                    "K",
```

Comment:
The auto-highlighting feature for options looks for the given shortcut in the description string. Since "K" is no in "Don't Verify Upstream Certs" this will fail. Pick a mnemonic that's in the string.